templates/comments_section.twig

```diff
@@ -1,16 +1,13 @@
-<div class="container">
+<div class="container page-section">
+	<h2 class="page-section-header">{{ __( 'Comments', domain ) }}</h2>
```

Comment:
I guess ` __( 'Comments', domain )` - here we need to use `__( 'Comments', 'planet4-master-theme' )` ?
Please confirm @koyan @angtheod

Reply:
Comments exists as a localised message in wordpress, so we don't need to retranslate it.

Reply:
But I am not sure if we get that by using "domain" there. Have you confirmed that @kirdia ?

Reply:
No, I haven't checked if the text domain was working properly, but you are right. We do not need to retranslate common words like 'Comments', 'Comment' etc. I will remove the planet4 master theme text domain for those.